pkg/config/config.go

```diff
@@ -70,6 +71,7 @@ type Config struct {
 	AdminPort  int
 	Fs         Fs
 	CouchDB    CouchDB
+	Redis      *redis.Options
```

Comment:
And it should be configurable from command-line (`cozy-stack serve --redis-cache redis://localhost:6379/0`)